src/main/java/com/acme/dbo/txlog/ByteCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.acme.dbo.txlog;
+
+import static java.lang.Math.abs;
+
+public class ByteCommand {
+        private static String DECOR = "primitive: ";
+
+        public Byte currentValue;
+        private static String accumulator;
+        private static Byte sum;
+        private static Byte MAX_BYTE = Byte.MAX_VALUE;
+
+        public ByteCommand(Byte message) {
+            currentValue = message;
+            if (accumulator == null) {
+                accumulator = message.toString();
+                sum = message;
+            }
+        }
+
+        public static String getDecoratedState() {
```

Comment:
Static methods ((( With OOP we don't use it. For static methods won't work inheritance and overriding so you can't extend your system through inheritance. You can't subclass your ByteCommand in future ((( And as side effect it has very bad testability. So if we move to OOP you have to get rid of static methods and use _object_ state and methods.

Reply:
I can't get rid of static methods. Removed all static methods and fields except static imports. Compiler keeps saying in Test files in rows:
 "        Facade.log(1);
        Facade.flush();"
Non-static method log (int) can't be referenced from static context. Where else is this static context setup?

Reply:
Sure you can ) We have made design decisions:
1. Facade stay with static methods for backward compatibility
2. New classes (e.g. Controller) we design by OOP way: it won't have any static methods
So your solution will look like:
`
class Facade {
  private static Control controller = new Controller(...anything it needs...); // <-- it's object!

  static void log(int message) {
    controller.log(new IntCommand(message)); // <-- you call _object_ method
  }
}
`

Reply:
Thank you! It worked!